db/migrate/20190924061752_create_levels.rb

```diff
@@ -4,8 +4,9 @@ def change
       t.belongs_to :rubric_criterion, foreign_key: true, null: false
       t.string :name, null: false
       t.integer :number, null: false
-      t.string :description, null: false
+      t.text :description, null: false
```

Comment:
In general you shouldn't make changes to an existing migration, but also you shouldn't make changes to a migration without also committing the new `schema.rb` file.

In this particular case, I'd say that you don't need to make these changes here, and instead should revert them and focus on the `Level` model file.